packages/ui-react/src/components/UserMenu/UserMenu.module.scss

```diff
@@ -32,10 +32,11 @@
 .sectionHeader {
   width: 100%;
   padding: 0 0 12px 24px;
-  color: variables.$slate-gray;
+  color: variables.$white;
```

Comment:
This is a change for the contrast benefit? I doubt if that works, because the opacity: 0.7 actually undoes this again.

Reply:
Slate grey is more of a blueish color (with `opacity: 0.7`). So together with Stephan we decided that adjusting this to the grey color that you can see in the other menu items was sufficient enough, see:

`slate-grey`
![Screenshot 2024-02-05 at 15 49 38](https://github.com/Videodock/ott-web-app/assets/48496458/5e3c1c18-f76b-412c-80d2-8d2774fb1184)